.travis.yml

```diff
@@ -27,6 +27,7 @@ addons:
 # Environment variables
 env:
   secure: "C0+Hlfa0YGErxUuWV00Tj6p45otC/D3YwYFuLpi2mj1rDFn/4dgh5WRngjvdDBVbXJ3duaZ78jPHWm1jr7vn2jqj9yETsCIK9psWd38ep/FEBM0SDr6MUD89OuXk/YyvxJAE+UXF6bXg7giey09g/CwBigjMW7ynET3wNAWPHPs="
+  secure: "I9n5trFOAFb323iwJtna5eKoRxwSDtzmyq8phhjwZcE/Ory+TM63wbSA7lL4I63ru4ydo064kvq7aLqVvV/dUENjG9tMbXdYRQgLJ1U+iIUr3awKKJy9WpsJuJMY7C7078sRja2HuC0bRtXSHrepmQO7GZ2Fi6MAPhq/MF2cyQI="
```

Comment:
I would strongly prefer that we defined this in the travis web interface (like we do for the macos packaging variables) instead of hardcoding it here. It is a PITA to have to cherry pick commits to override these secure variables whenever someone wants to create test builds on their local fork, and I have been meaning to file a PR to remove the one above for some time now.

Reply:
Removed it. The duplicated variable was also an invalid build configuration according to https://travis-ci.org/github/OpenRA/OpenRA/jobs/693947105/config